ci-operator/step-registry/hypershift/install/hypershift-install-commands.sh

```diff
@@ -60,7 +65,6 @@ fi
 if [ "${CLOUD_PROVIDER}" == "Azure" ]; then
   "${HCP_CLI}" install --hypershift-image="${OPERATOR_IMAGE}" \
   --enable-conversion-webhook=false \
-  --managed-service=ARO-HCP \
   --external-dns-provider=azure \
```

Comment:
This this going to default ARO-HCP on the backside?

https://github.com/openshift/hypershift/blob/8d227d69b4b54fe5ba86bab93c7ffe271d982782/cmd/install/install.go#L171-L173

Reply:
Nope the intention here was to remove the managed-service flag when the `DISABLE_ARO_MANAGED_SERVICE` var was set to true. Looking at the code you mentioned we should set the managed-serice flag to an empty string rather than removing it entirely. I amended the commit with the required change.